src/components/Common/VideoCard/video-card.component.jsx

```diff
@@ -1,19 +1,28 @@
 import React, { Fragment } from 'react';
 import { Link } from 'react-router-dom';
-import formatter from '../../../utils/formatter';
+import helper from '../../../utils/helpers';
 import { CardMutedText, TopImage, VideoCardWrapper } from './Styled/styled-components';
 
 const VideoCard = (props) => {
-  const { thumbnalURL, title, channelTitle, publishTime } = props;
+  const { thumbnalURL, title, channelTitle, publishTime, index, videoId, onClick } =
```

Comment:
Seems like this component and the `VideoCardFlat` are the same or really alike, with some CSS differences, you could have a prop to change between top image and side image, or you could pass a prop to the styled component to change between the two